tests/lib/prepare.sh

```diff
@@ -471,6 +480,9 @@ EOF
     # - make sure the group matches
     # - bind mount /root/test-etc/* to /etc/* via custom systemd job
     # We also create /var/lib/extrausers/* and append ubuntu,test there
+    ! test -e /mnt/system-data/root
+    mkdir -m 700 /mnt/system-data/root
+    test -d /mnt/system-data/root
```

Comment:
This reminded me that for many years Ubuntu shipped an 0755 /root directory. It's possible that other distros might do this today. That obviously doesn't affect this hunk of the test so just fyi.